cypress/e2e/UnitTestMenu.cy.js

```diff
@@ -0,0 +1,53 @@
+// cypress/e2e/login_and_menu.cy.ts
+describe('Login y verificación de menú', () => {
```

Comment:
should we divide into separate folders unit tests and integration tests instead of leaving them all inside `e2e`? what do you think?